src/components/navigation-header/style.module.css

```diff
@@ -33,20 +33,36 @@
   list-style: none;
   margin-bottom: 0;
   margin-left: 0;
-  margin-right: 56px;
+  margin-right: 72px;
   margin-top: 0;
 
   & li {
-    padding: 8px 12px;
+    padding: 6px 8px;
```

Comment:
Looks like our padding is off a bit from design. Looks a little shorter on the y axis.

<img width="1051" alt="CleanShot 2021-12-15 at 16 36 45@2x" src="https://user-images.githubusercontent.com/825855/146268839-7c4e6ac8-b680-4622-ad39-4a92c12b47cb.png">

![CleanShot 2021-12-15 at 16 36 18@2x](https://user-images.githubusercontent.com/825855/146268847-083ab90c-e6ed-4b66-b6d4-cbd74c7d4cea.png)

Reply:
Ah, I was looking at the wrong page in designs. I'll address this!

Reply:
Addressed! https://github.com/hashicorp/dev-portal/pull/45/commits/73753bc5ecf8cf626fe70554e6f575fdfa490adf